src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -92,7 +95,7 @@ internal TestRunner(Options options, ITestExecutor testExecutor)
                     running.Add(task);
                 }
 
-                Console.WriteLine($"  { running.Count} running, { waiting.Count} queued, { completed.Count} completed");
+                Console.WriteLine($"  { running.Count, 2} running, { waiting.Count,2} queued, { completed.Count,2} completed{((failures <=0)?"":$", {failures,2} failures")}");
```

Comment:
Also, we should be a bit more consistent on whitespace surrounding literals and operators here.